packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -204,15 +269,20 @@ export default decorate([
         />
         <br />
       </CardHeader>
-      <CardBlock className='text-center'>
-        <div>
-          <span className='text-muted'>{_('os')}</span> <strong>{os}</strong>
-        </div>
-        <div>
-          <span className='text-muted'>{_('version')}</span>
-          {'  '}
-          <strong>{version}</strong>
-        </div>
+      <CardBlock>
+        <div
+          className='text-muted'
+          dangerouslySetInnerHTML={{
+            __html: marked(defined(state.description)),
```

Comment:
`defined` has not an utility in this case. I want to say to create a  computed which returns `{ __html: marked(defined(description, _description))}` It will avoid multiple parse.